api/api.go

```diff
@@ -580,6 +580,36 @@ func (api *api) GetAutoSwapsConfig() (*GetAutoSwapsConfigResponse, error) {
 	}, nil
 }
 
+func (api *api) InitiateSwap(ctx context.Context, initiateSwapRequest *InitiateSwapRequest) error {
```

Comment:
I think we should use different endpoints for swapping in and out, to reduce complexity

Reply:
Done, now we have three endpoints
`/api/wallet/swap` -> for auto swaps ("GET", "POST", "DELETE")
`/api/wallet/swap-out` -> "POST"
`/api/wallet/swap-in` -> "POST" (to be added)